Remove highcharts

janezd opened the pull request:
##### Issue

#111, #112 and #113 removed dependencies on highcharts. This PR removes it from distribution.

~~Requires #112 and #113.~~

PrimozGodec commented:
Now when we removed the Highcharts we can also change the license of this add-on (I think the current CC was set because we used HighCharts). @janezd do you know anything about licenses? We have different licenses for addons and Orange. I think GPL is the most common for addons.

janezd commented:
I think you're right: we switched to Creative Commons because of (or at least at the same time as when we added) highcharts. GPL would be more appropriate, also because of PyQt --- I'm not sure whether and how this goes together.

I changed the license.
